setup.py

```diff
@@ -12,7 +12,7 @@
     long_description=open('README.rst').read(),
     author=djcroco.__author__,
     author_email='matt.lenc@gmail.com',
-    url='https://github.com/mattack108/djcroco/',
+    url='https://github.com/incuna/django-croco/',
```

Comment:
I think we should remove the Author's email -- I suspect that he'd prefer not to hear any more about this particular project.